lib/generators/deimos/schema_class/templates/schema_enum.rb.tt

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# This file is autogenerated by Deimos, Do NOT modify
+module Deimos
+  # :nodoc:
+  class <%= Deimos::SchemaBackends::AvroBase.schema_classname(@current_schema) %> < SchemaClass::Enum
+    # @return ['<%= @current_schema.symbols.join("', '") %>']
+    attr_accessor :<%= @current_schema.name.underscore %>
+
+    # :nodoc:
+    def initialize(<%= @current_schema.name.underscore %>)
+      super()
+      self.<%= @current_schema.name.underscore %> = <%= @current_schema.name.underscore %>
```

Comment:
see above conversation, what happened there?

Reply:
Removing these causes breakages !! - Discussion is below!